OxfordBodleian/Ullendorff/BDLeu9.xml

```diff
@@ -34,9 +37,81 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                ስዕሉ<hi rend="rubric" rendition="#partialRubric">፨</hi> 
+                                <seg type="supplication">አድኀና፡ ወዕ<add place="interlinear">ቀ</add>ባ<hi rend="rubric" rendition="#partialRubric">፨</hi> ለአመትከ<hi rend="rubric" rendition="#partialRubric">፨</hi> <persName role="owner" ref="PRS14023WalattaM"><add place="inline">ወለተ፡ ሚካኤል፡ 
+                                    በተሐ።</add> በተሐ፡ በተሃ</persName></seg>
+                            </incipit> 
+                          
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="1r26" to="1r41"/>
+                            <title type="complete" ref="LIT6830PPAyna"/>
+                            <textLang mainLang="gez"/>
+                            <!-- this is the entire text-->
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡</hi> ፩ አምላክ፡ ጸሎት። በእንተ<hi rend="rubric" rendition="#partialRubric">፨</hi> ዓይነ፡ ጥላ፡ ወዓይነ፡ ዛር፡ ወሾተላይ፡ 
+                                <hi rend="rubric">ኢየሱስ፡</hi> ናዝራዊ<hi rend="rubric" rendition="#partialRubric">፨</hi> <sic>በትምርተ<supplied reason="omitted">፡</supplied></sic> መስቀል<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                መርአዊ፡ ጸረ፡ ፈውስ<supplied reason="omitted">፡</supplied> ሲቃ<hi rend="rubric" rendition="#partialRubric">፨</hi> ጳጢቃ<hi rend="rubric" rendition="#partialRubric">፨</hi> ኢዮሎን<hi rend="rubric" rendition="#partialRubric">፨</hi> አኽያ፡ ሾራኽያ<hi rend="rubric" rendition="#partialRubric">፨</hi> አልሸይ<hi rend="rubric" rendition="#partialRubric">፨</hi> ኢማኑኤል<hi rend="rubric" rendition="#partialRubric">፨</hi> 
+                                ለእግዚአብሔር<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                <seg type="supplication">አድኅና<hi rend="rubric" rendition="#partialRubric">፨</hi> ለአመትከ፡ <persName ref="PRS14023WalattaM" role="owner"><add place="inline">ወለተ፡ ሚካኤል፡ በተሐ፡</add> በተሐ፨</persName></seg>
+                            </incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i3">
+                            <locus from="1r41" to="1r76"/>
+                            <title type="complete" ref="LIT5920JohnBeg"/>
+                            <textLang mainLang="gez"/>
+                            <!-- this is the entire text-->
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric"><sic>ቀ</sic> በስመ፡ <add place="interlinear">ወ</add>ወልድ፡ ወመንፈ</hi>ስ፡ ቅዱስ፡
```

Comment:
Since this is a well-established formula, I would add `<supplied reason="omitted">አብ፡</supplied>` after በስመ፡